bitmap-ittybitty/lib/handler.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+const fs = require('fs');
+
+
+function reader(path, cb) {
```

Comment:
Taking the time to name your functions something super descriptive is worth it's weight in gold. This describes what it does, but not very well. I tend to name my functions absurdly long things, which you don't have to do, but because of tab completion, there's no real obstacle to writing descriptive names. A good, simple name for this would be something like readBitmap.